lib/dialects/postgres/connection-manager.js

```diff
@@ -209,6 +209,11 @@ class ConnectionManager extends AbstractConnectionManager {
       query += 'SET standard_conforming_strings=on;';
     }
 
+    if (this.sequelize.options.clientMinMessages !== undefined) {
+      console.warn('Usage of "options.clientMinMessages" is deprecated and will be removed in v7.');
+      console.warn('Please use "option.dialectOption.clientMinMessages" instead.');
```

Comment:
```suggestion
      console.warn('Please use "option.dialectOptions.clientMinMessages" instead.');
```

Reply:
Oh I copied this from the main branch. There probably is the same typo